### PR TITLE
Add Datadog monitoring configuration

### PR DESCRIPTION
**Pull request by awang**
Explain how to setup Datadog monitoring

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/3374)

### CODE REVIEW

File: doc/user/content/monitoring/_index.md [hunk 1]
```diff
@@ -70,6 +70,10 @@ Materialize provides a [recommended dashboard][dashboard-json] that you can [imp
 Grafana][graf-import]. It relies on you having configured prometheus to scrape
 materialized.
 
+## Datadog
+
+Materialize metrics can be sent to Datadog via the [OpenMetrics agent check](https://www.datadoghq.com/blog/monitor-prometheus-metrics/).  (Requires Datadog Agent 6 and above). Simply configure _prometheus_url_ (ie `http://<materialized host>/metrics`), namespace, and metrics (ie `mz*`) in _openmetrics.d/conf.yaml_.
```
Comment (quodlibetor) [inline doc/user/content/monitoring/_index.md:75]: We don't lint on this so TIOLI but most of these files have a line length of at most 100 characters, and it is slightly nice to keep to that.

Comment (quodlibetor) [reply doc/user/content/monitoring/_index.md:75]: Also I think both "prometheus_url" and the conf file possibly should be in quotes? I think that there are genuine philosophies around this and I'm not sure that I'm right, your call.

Comment (awang) [reply doc/user/content/monitoring/_index.md]: makes sense!  i updated based on both of those recommendations

**Comment by quodlibetor** [comment]
The build is failing because of trailing whitespace 〿

**Pull request merged**